TYPO3.Media/Classes/TYPO3/Media/Controller/AssetController.php

```diff
@@ -607,7 +607,7 @@ protected function maximumFileUploadSize()
     public function addFlashMessage($messageBody, $messageTitle = '', $severity = Message::SEVERITY_OK, array $messageArguments = array(), $messageCode = null)
     {
         if (is_string($messageBody)) {
-            $messageBody = $this->translator->translateById($messageBody, $messageArguments, null, null, 'Main', 'TYPO3.Media') ?: $messageBody;
+            $translatedMessageBody = $this->translator->translateById($messageBody, $messageArguments, null, null, 'Main', 'TYPO3.Media') ?: $messageBody;
```

Comment:
Hey aske, /me confused ;)

Why should this match 2.2? It's a PR to master fixing https://github.com/neos/media/commit/427d0ed70e5365fa906f4196c936e4c9271d266e#diff-49588a4d4cf59c679cf744ce4dc47725 , which is in master right?